app/models/server/models/LivechatInquiry.js

```diff
@@ -28,7 +28,9 @@ export class LivechatInquiry extends Base {
 			},
 			{
 				sort: {
-					ts: 1,
+					queueOrder: 1,
+					estimatedWaitingTimeQueue: 1,
+					estimatedServiceTimeAt: 1,
```

Comment:
Need to index those fields

Reply:
Okay. I created [compound Indexes](https://docs.mongodb.com/manual/core/index-compound/#index-type-compound) for this purpose, following the mongodb docs.